### DISCUSSION TITLE
Next Js 13 how to turn off output standalone on Amplify

### DISCUSSION THREAD

**Discussion by vyshak-sukumaran** [vercel/next.js#47073] [Help]
### Summary

CustomerError: Standalone directory not found in /codebuild/output/src131660783/src/<dir_name>/.next/standalone. Please enable output standalone on your next.config.js file or set NEXT_PRIVATE_STANDALONE=true. https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files

Please, any help will be appreciated.

### Additional information

```js
I am deploying my application on amplify and stuck with this error. Is there a way to turn off output standalone feature ?
```

**Comment by philwolstenholme** [comment]
What have you tried so far? 

I think Amplify _needs_ the standalone output turned on, that output is what it deploys. Have you tried following the advice in the error message or on https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files?

**Reply by vyshak-sukumaran** [reply]
Yes, I've.

```
const nextConfig = {
  reactStrictMode: true,
  output: 'standalone',
}

module.exports = nextConfig


```
This is my config file.
